python/ray/tests/test_object_spilling.py

```diff
@@ -51,6 +51,14 @@ def assert_no_thrashing(address):
         f"consumed: {consumed_bytes}, restored: {restored_bytes}")
 
 
+def assert_no_worker_blocking():
```

Comment:
This check is a little sketchy. Is there a way we can have the test fail on timeout instead? I think it would work if we could increase the worker startup timeout to something very high.

Reply:
done